bundle.Dockerfile

```diff
@@ -4,8 +4,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=service-telemetry-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=latest
-LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,stable-1.1
```

Comment:
This PR targets the wrong branch. We should not be releasing the master branch as "stable"

Reply:
Resolved